packages/dataprovider/src/buildGqlQuery.ts

```diff
@@ -215,6 +220,24 @@ export default (introspectionResults: IntrospectionResult) => (
     ? buildFieldsFromFragment(fragment, resource.type.name, aorFetchType)
     : buildFields(introspectionResults)((resource.type as any).fields);
 
+  const totals: Record<QueryDialect, FieldNode> = {
+    "nexus-prisma": gqlTypes.field(gqlTypes.name(countName), {
+      alias: gqlTypes.name("total"),
+      arguments: supportsCountArgs ? countArgs : undefined,
+    }),
+    typegraphql: gqlTypes.field(gqlTypes.name(`${queryType.name}Aggregate`), {
+      alias: gqlTypes.name("total"),
+      arguments: supportsCountArgs ? countArgs : undefined,
+      selectionSet: gqlTypes.selectionSet([
+        gqlTypes.field(gqlTypes.name("count"), {
+          selectionSet: gqlTypes.selectionSet([
+            gqlTypes.field(gqlTypes.name("_all")),
+          ]),
+        }),
+      ]),
+    }),
+  };
```

Comment:
maybe enclose the stuff that is only required for `nexus-prisma` (like countName and supportsCountArgs)